docs/src/views/FocusZone.tsx

```diff
@@ -0,0 +1,366 @@
+        <b>direction</b> - Defines which arrows to react to.
+        <p>Type: {code('FocusZoneDirection')}, enum with next options:</p>
+        <ul>
+          <li>
+            <i>horizontal</i> - navigation between items can be made using left/right arrow keys.
+            <p>See an example of {link('horizontal menu', '/components/menu#types-menu')}.</p>
+          </li>
+          <li>
+            <i>vertical</i> - navigation between items can be made using up/down arrow keys.
+            <p>
+              See an example of{' '}
+              {link('vertical menu', '/components/menu#variations-vertical-pointing')}.
+            </p>
+          </li>
+          <p>
+            Vertical and horizontal menu share same accessibility behavior and direction is defined
+            by prop "vertical":
+          </p>
+          <CodeSnippet
+            label="menuBehavior.ts"
+            value={`
+              const menuBehavior: Accessibility = (props: any) => ({
+                //...
+                focusZone: {
+                  mode: FocusZoneMode.Embed,
+                  props: {
+                    direction: props.vertical ?
+                    FocusZoneDirection.vertical :
+                    FocusZoneDirection.horizontal,
+                    //...
+                  },
+                },
+              })
+              `}
+          />
+          <li>
+            <i>bidirectional</i> - navigation between items can be made using all arrow keys.{' '}
+            <b>It is set by default.</b>
+            <p>See an example of {link('tab list', '/components/menu#usages-tab-list')}.</p>
+          </li>
+        </ul>
+      </li>
+      <li>
+        <b>defaultTabbableElement</b> - Function which uses root element as parameter to return the
+        initial active element. For example, when there is a chat with a bottom-up approach, it is
+        needed the last chat message to be tabbable (active), not the first one, which is by
+        default.
+        <p>Type: {code('(root: HTMLElement) => HTMLElement')}</p>
+        <CodeSnippet
+          label="chatBehavior.ts"
+          value={`
+          const chatBehavior: Accessibility = (props: any) => ({
+            //...
+            focusZone: {
+              //...
+              props: {
+                defaultTabbableElement: (root: HTMLElement) => {
+                  return root.querySelector('[data-last-visible="true"]');
+                },
+              },
+            },
+          })
+          `}
+        />
+      </li>
+      <li>
+        <b>shouldFocusOnMount</b> - If a default tabbable element should be force focused on
+        FocusZone mount.
+        <p>Default: {code('false')}</p>
+        <p>Type: {code('boolean')}</p>
+      </li>
+      <li>
+        <b>shouldFocusInnerElementWhenReceivedFocus</b> - if true and FocusZone's root element
+        (container) receives focus, the focus will be forced to defaultTabbableElement (if set) or
```

Comment:
... the focus will land either on the defaultTabbableElement (if set) or on the first tabbable ...